src/server.c

```diff
@@ -5193,13 +5211,19 @@ void getKeysSubcommand(client *c) {
 void commandCommand(client *c) {
     dictIterator *di;
     dictEntry *de;
+    int numcmds = 0;
 
-    addReplyArrayLen(c, dictSize(server.commands));
+    void *replylen = addReplyDeferredLen(c);
```

Comment:
we need to patch commandCountCommand too

Reply:
OK, will modify it to loop over the commands and count internal ones for internal connections only as well.

Reply:
Added the unified version from RL